net/icmpv6/icmpv6_reply.c

```diff
@@ -112,15 +112,19 @@ void icmpv6_reply(FAR struct net_driver_s *dev, int type, int code, int data)
 
   dev->d_len = ipicmplen + datalen;
 
+  /* Copy fields from original packet */
+
+  memmove(icmpv6 + 1, ipv6, datalen);
```

Comment:
Why to skip a byte in the icmpv6 destination?

Reply:
It's skipping the whole header, to copy origin packet as the payload of icmpv6.
It's removed by accident and I'm just restoring the code from [previous version](https://github.com/apache/nuttx/blob/releases/11.0/net/icmpv6/icmpv6_reply.c#L117-L119).